src/main/java/Game.java

```diff
@@ -10,11 +16,11 @@ public void setWinner(Player winner) {
         this.winner = winner;
     }
 
-    public int getNumOfPlayers() {
-        return numOfPlayers;
+    public void addPlayer(Player player) {
```

Comment:
Just realized, will we be adding players one by one or by passing an array of names from the BingoApp class?

Reply:
Yes, this adds players one by one.